common/src/main/java/com/wynntils/wynn/model/CharacterSelectionManager.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model;
+
+import com.wynntils.core.managers.CoreManager;
+import com.wynntils.gui.screens.CharacterSelectorScreen;
+import com.wynntils.mc.event.ContainerSetContentEvent;
+import com.wynntils.mc.event.MenuEvent;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.wynn.objects.ClassInfo;
+import com.wynntils.wynn.objects.ClassType;
+import com.wynntils.wynn.utils.ContainerUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+public class CharacterSelectionManager extends CoreManager {
+    private static final Pattern NEW_CLASS_ITEM_NAME_PATTERN = Pattern.compile("§a\\[+\\] Create new character");
```

Comment:
This pattern is missing some parts

```suggestion
    private static final Pattern NEW_CLASS_ITEM_NAME_PATTERN = Pattern.compile("§l§a\\[\\+\\] Create a new character");
```

Reply:
Thanks. I could never test this as I have max class count.